File: hw13/testfacts.c
#include <stdio.h>

int factorial(int n) {
    if (n == 0) {
        return 1;
    }
    else {
        return n * factorial(n-1);
    }
}

int main() {
    int funcvalue;
    funcvalue = factorial(10);
    printf("%d", funcvalue);
}

File: hw13/testfacts.py
def factorial(arg):
  arg = int(arg)
  if arg > 1:
    return arg * factorial(arg-1)
  else:
    return arg

# userinput = input("What do you want to factorial?\n")
# Commented out user input capabilities
userinput = 10
print(factorial(userinput))